From 8ecb4788f256756c27524a0a56ba24ce23244126 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C3=86var=20Arnfj=C3=B6r=C3=B0=20Bjarmason?= <avarab@gmail.com>
Date: Tue, 7 Sep 2010 11:03:45 +0000
Subject: [PATCH 01/20] gettextize: git-clean clean.requireForce braces
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Use braces around a nested if/if/else to appease GCC, which'll whine
under -Wall and "suggest explicit braces to avoid ambiguous ‘else’".

This code was originally added in "gettextize: git-clean
clean.requireForce messages", and tested under clang (which doesn't
whine about perfectly valid uses of C).

Signed-off-by: Ævar Arnfjörð Bjarmason <avarab@gmail.com>
---
 builtin/clean.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/builtin/clean.c b/builtin/clean.c
index 50ade84..52ec17a 100644
--- a/builtin/clean.c
+++ b/builtin/clean.c
@@ -77,13 +77,14 @@ int cmd_clean(int argc, const char **argv, const char *prefix)
 	if (ignored && ignored_only)
 		die(_("-x and -X cannot be used together"));
 
-	if (!show_only && !force)
+	if (!show_only && !force) {
 		if (config_set)
 			die(_("clean.requireForce set to true and neither -n nor -f given; "
 				  "refusing to clean"));
 		else
 			die(_("clean.requireForce defaults to true and neither -n nor -f given; "
 				  "refusing to clean"));
+	}
 
 	if (force > 1)
 		rm_flags = 0;
-- 
1.7.2.3.313.gcd15

